plasoscaffolder/frontend/frontend.py

```diff
@@ -0,0 +1,373 @@
+    """Return the path to the project as chosen by the user.
+
+    Args:
+      definition (definition_interface.ScaffolderDefinition): the chosen
+          definition. Used to validate the project path.
+
+    Returns:
+      str: the path to the project file.
+
+    Raises:
+      errors.WrongCliInput: when no valid project path has been provided.
+    """
+    cls.OUTPUT_HANDLER.PrintNewLine()
+    # pylint: disable=assignment-from-no-return
+    project_path = cls.OUTPUT_HANDLER.PromptInfo('Path to the project root')
+    if definition.ValidatePath(project_path):
+      cls.OUTPUT_HANDLER.PrintOutput(
+          'Path [{0:s}] set as the project path.'.format(project_path))
+      return project_path
+
+    # pylint: disable=assignment-from-no-return
+    check = cls.OUTPUT_HANDLER.Confirm((
+        'Path [{0:s}] does not lead to a valid project for {1:s}. '
+        'Do you want to try again?').format(project_path, definition.NAME))
+
+    if check:
+      return cls.GetProjectPath(definition)
+
+    raise errors.WrongCliInput(
+        u'Unable to proceed without a valid project path.')
+
+  @classmethod
+  def GetScaffolder(
+      cls, definition: definition_interface.ScaffolderDefinition
+  ) -> scaffolder_interface.Scaffolder:
+    """Return the scaffolder as chosen by the user.
+
+    Args:
+      definition (definition_interface.ScaffolderDefinition): the chosen
+          definition. Used to determine available scaffolders.
+
+    Returns:
+      scaffolder_interface.ScaffolderCli: the chosen scaffolder object.
+    """
+    scaffolders = {}
+    get_scaffolders = scaffolder_manager.ScaffolderManager.GetScaffolders
+    for scaffolder_name, scaffolder in get_scaffolders():
+      if scaffolder.PROJECT == definition.NAME:
+        scaffolders[scaffolder_name] = scaffolder
+
+    cls.OUTPUT_HANDLER.PrintNewLine()
+    cls.OUTPUT_HANDLER.PrintInfo(
+        'Available scaffolders for {0:s}:'.format(definition.NAME))
+
+    scaffolder = cls._GetSelection(list(scaffolders.keys()), 'Scaffolder')
+    if scaffolder in scaffolders:
+      return scaffolders[scaffolder]()
+
+    cls.OUTPUT_HANDLER.PrintError(
+        'Scaffolder: {0:s} does not exist.'.format(scaffolder))
+    return cls.GetScaffolder(definition)
+
+  @classmethod
+  def Start(cls, definition_value):
+    """Start the CLI.
+
+    Args:
+      definition_value (str): the definition string chosen by UI.
+    """
+    cls.OUTPUT_HANDLER.PrintInfo('   == Starting the scaffolder ==')
+    cls.OUTPUT_HANDLER.PrintInfo('Gathering all required information.')
+    scaffolder_engine = engine.ScaffolderEngine()
+
+    definition = cls.GetDefinition(definition_value)
+
+    project_path = cls.GetProjectPath(definition)
+    scaffolder_engine.SetProjectRootPath(project_path)
+
+    module_name = cls.GetModuleName()
+    scaffolder_engine.SetModuleName(module_name)
+    try:
+      cls.CreateGitFeatureBranch(project_path, scaffolder_engine.module_name)
```

Comment:
I think we should let the user know we're going to do this, can you add something to that effect?